lib/ts/PriorityQueue.h

```diff
@@ -110,6 +110,12 @@ PriorityQueue<T, Comp>::pop()
     return;
   }
 
+  // SKH - I suspect this assignment is not preserving entry indices correctly.
```

Comment:
Ok.  Hopefully a transient comment anyway.  @masaori335 should be able to take a look and tell me whether this is a concern or not.